src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +129,10 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates.stream()
+                .filter(candidateValidator)
```

Comment:
```suggestion
                .filter(new CandidateValidator())
```
and you can remove initialization in the constructor

Reply:
Okay. I see your comment in the PR of Olena. Both variants are possible. You can skip this comment.